config/autoscaling/development-policy.json

```diff
@@ -27,28 +19,20 @@
       "adjustment": "-1"
     },
     {
-      "metric_type": "memoryutil",
-      "breach_duration_secs": 120,
+      "metric_type": "cpu",
+      "breach_duration_secs": 60,
       "threshold": 50,
       "operator": ">=",
       "cool_down_secs": 120,
-      "adjustment": "+1"
+      "adjustment": "+2"
     },
     {
       "metric_type": "cpu",
-      "breach_duration_secs": 120,
-      "threshold": 60,
+      "breach_duration_secs": 60,
+      "threshold": 5,
       "operator": "<",
-      "cool_down_secs": 120,
+      "cool_down_secs": 60,
```

Comment:
Dev is using very similar metrics to staging and prob with the following differences:

- Less instances overall (min and max) 
- More reactive to scale down to facilitate stress testing.